it/repositories/ReactiveRepositoryHelpersSpec.scala

```diff
@@ -0,0 +1,67 @@
+package repositories
+
+import model.Exceptions.{ NotFoundException, NotModifiedException }
+import play.api.libs.json.Json
+import reactivemongo.bson._
+import reactivemongo.api.DB
+import reactivemongo.bson.{ BSONDocument, BSONObjectID }
+import testkit.MongoRepositorySpec
+import uk.gov.hmrc.mongo.ReactiveRepository
+import uk.gov.hmrc.mongo.json.ReactiveMongoFormats
+
+class ReactiveRepositoryHelpersSpec extends MongoRepositorySpec {
```

Comment:
The singleRemovalValidator is only being used in dormant code, but I have now added the test for the future